docs/how-to-guides/integrating-autoware/creating-vehicle-interface-package/creating-vehicle-interface-for-ackerman-kinematic-model.md

```diff
@@ -1,5 +1,198 @@
-# Creating vehicle interface for ackerman kinematic model
+# Creating vehicle interface for Ackermann kinematic model
 
-!!! warning
+This page introduces a module vehicle_interface and explains how to implement it.
 
-    Under Construction
+## What is a vehicle interface
+
+`vehicle interface` is an interface that connects the control commands and your hardware.  
```

Comment:
The term "vehicle ECU" is specific to road vehicles and less relevant to robots applicable to Autoware. (Autoware is mainly focused on cars though.) How about "control devices"?

Reply:
I believe the word `vehicle` is a generic one, and every vehicle has an ECU.
But rephrasing sounds good to me. I prefer `your vehicle's control device` because `your control device` is vague.